src/controllers/AdminController.php

```diff
@@ -334,12 +336,45 @@ class="fb-cta cta--yellow"
     $timer_on = $timer->getValue() === '1';
     $timer_off = $timer->getValue() === '0';
 
-    if ($start_ts->getValue() === '0') {
-      $start_ts = tr('Not started yet');
-      $end_ts = tr('Not started yet');
+    if ($start_ts->getValue() !== '0' && $start_ts->getValue() !== 'NaN') {
```

Comment:
Yes, "NaN" (not a number) can be returned via the client (JavaScript). However, this should not happen with the current client-side code. It was included as a fail-safe.